manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
+PackageLocale: en-US
+Publisher: BlastApps
+PackageName: Fluent Search

```

Comment:
```suggestion
PackageName: Fluent Search Beta
```

Reply:
I'm not sure how appropriate this is. It's still in the description on the Windows store and the Version Number is <1, but I think it does not feel like beta software. Maybe @adirh3 could weigh in, but it's probably not a big deal either way.

Reply:
According to the latest docs over at https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#packagename, the name has to match with what's shown in Control Panel.

If it's out of beta, you'll have to contact the application's developer to update the name.

Reply:
Oh, alright. He's contacted me privately, the Beta name is still correct, sorry for any inconveniences on your end, you've done most of the work here.